src/features/DisplayControls/NursingTasks/components/NursingTasks.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { FormattedMessage } from "react-intl";
+import "../styles/NursingTasks.scss";
+
+import {
+  fetchMedicationNursingTasks,
+  ExtractMedicationNursingTasksData,
+  GetUTCEpochForDate,
+} from "../utils/NursingTasksUtils";
+import TaskTile from "./TaskTile";
+
+export default function NursingTasks(props) {
+  const { patientId } = props;
+  const [medicationNursingTasks, setMedicationNursingTasks] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const NoNursingTasksMessage = (
+    <FormattedMessage
+      id={"NO_NURSING_TASKS_MESSAGE"}
+      defaultMessage={"No nursing task is scheduled for the patient"}
+    />
+  );
+
+  useEffect(() => {
+    const fetchNursingTasks = async () => {
```

Comment:
can you please move this method out of useEffect, just have the invoke inside useEffect